cli/internal/server/server.go

```diff
@@ -0,0 +1,169 @@
+package server
```

Comment:
`daemon : server :: ____ : client`

a. `TurboClient`
b. `daemonclient`
c.
d.
e. None of the above.

I can trace what it's doing; I've built my mental model on the functionality not the names. I think we'd be well-served by spending a few minutes spitballing names.

Reply:
I'm definitely open to suggestions on names. Some of where this is currently is fairly path-dependent. The one caveat I'll add is that Client and Server come from protobuf/grpc: we will always have some `FooServer` and `FooClient` as long as we have a service in a proto file called `Foo`.